### ISSUE TITLE
[FIX] Prepend used environment variables with PROTEUS_

### ISSUE THREAD

**Issue by matt035343** [SneaksAndData/adapta#49]
**Is your feature request related to a problem? Please describe.**
Avoid conflicts with other environment variables in the system.

**Describe the solution you'd like**
Prepend used environment variables in this package with `PROTEUS_`.

**Comment by george-zubrienko** [comment]
I think it is better we extract the env var prefix globally and then apply it to all vars through a single function - i'll prepare a PR.

**Comment by matt035343** [comment]
So make it changeable for the user?

**Comment by george-zubrienko** [comment]
> So make it changeable for the user?

No, user will be able to use either `ADAPTA__` or `PROTEUS__` vars until 3.0, and then `ADAPTA__` only for >=3.x.

Idea is simple, instead of doing `os.environ` in we should do smth like this:
```
LEGACY_PREFIX = 'PROTEUS__'
PREFIX = 'ADAPTA__'

def get_configuration(name: str) -> Optional[str]:
    return os.getenv(f'{LEGACY_PREFIX}{name}') or os.getenv(f'{PREFIX}{name}')
```

and then replace all `os.getenv` etc calls with this method internally.
